svelte-kit-scss/src/lib/components/shared/Dropdown/DropdownMenuItemLayout.svelte

```diff
@@ -0,0 +1,18 @@
+<div class="dropdown-menu-item-layout">
+  <slot />
```

Comment:
I feel like this didn't necessarily need it's own component, it could just be in `DropdownMenuSelect.svelte`